Detectors/TPC/calibration/src/CorrectionMapsLoader.cxx

```diff
@@ -64,6 +64,7 @@ void CorrectionMapsLoader::addOptions(std::vector<ConfigParamSpec>& options)
 {
   addOption(options, ConfigParamSpec{"corrmap-lumi-mean", VariantType::Float, 0.f, {"override TPC corr.map mean lumi (if > 0), disable corrections if < 0"}});
   addOption(options, ConfigParamSpec{"corrmap-lumi-inst", VariantType::Float, 0.f, {"override instantaneous CTP lumi (if > 0) for TPC corr.map scaling, disable corrections if < 0"}});
+  addOption(options, ConfigParamSpec{"corrmap-lumi-scalemode", VariantType::Int, 0, {"override setting for scaling behaviour when using derivative maps"}});
```

Comment:
In the case of `scalemode==1` instead of the current static maps one should use derivative maps, right? Are they supposed to be extracted on the fly from TPC/Calib/CorretionMap or this will be a new object with an alternative path?
In the latter case instead of the device-level `corrmap-lumi-scalemode` option, one should have a workflow option (similar to https://github.com/AliceO2Group/AliceO2/blob/a7beabd472e4e8e44f6a7e1c51efe83788342180/GPU/Workflow/src/gpu-reco-workflow.cxx#L133)
since it must affect the inputs added by the helper in https://github.com/AliceO2Group/AliceO2/blob/a7beabd472e4e8e44f6a7e1c51efe83788342180/Detectors/TPC/calibration/src/CorrectionMapsLoader.cxx#L55.